index.html

```diff
@@ -14,7 +14,46 @@
   <header class="header-container">
     <a class="mylogo" href="#">My Logo</a>
     <nav class="menu">
-      <img src="image/menu.png" alt="Menu Icon">
+      <img src="icons/menu.png" alt="Menu Icon">
     </nav>
   </header>
-</body>
\ No newline at end of file
+  <main class="home">
+    <section class="bio">
+      <h1 class="title">I’m Jawad <br>
+        Glad to see you!</h1>
```

Comment:
- [ ] While you did a great job I see that you put a <be> tag in the ```h1``` to wrap the text, but if you see in the Figma that there is no ```br``` tag in the text, so try not to break your name just make it like that ``` >I’m Jawad``` at one line 😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/220176213-8441f87f-593b-434d-adad-0de65c136292.png)   |   ![image](https://user-images.githubusercontent.com/83360636/220176175-c985ffe7-fec2-4552-9a57-11b77e48fd77.png)    |

Reply:
@osamaashraf6 ,

Thank you. I have removed the <br> tag.